src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -8,6 +8,7 @@
 import org.openqa.selenium.WebElement;
 
 public class AdsFandomObject extends AdsBaseObject {
+  private final String FANDOM_FEED_SELECTOR = ".feed-container";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "FANDOM_FEED_SELECTOR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)